src/twisted/protocols/basic.py

```diff
@@ -403,6 +403,9 @@ def _handleParseError(self):
         self.brokenPeer = 1
 
 
+_MAX_LINE_LENGTH = 32768
```

Comment:
Even if this is a private variable, I think that it helps to add a comment about the scope/purpose of this variable.

it should help other know if they can reuse it, or define a new one.